docs/data-formats.md

```diff
@@ -224,22 +224,107 @@ are applied.
 
 ## Custom Attributes Annotations Format
 
-```json
-{
-  "[assembly-name]": {
-    "[namespace-name]": {
-      "[type-name]": {
-        "[field-or-property-name]": {
-          "[attribute-name]": "[attribute-value]"
-        },
-
-        "[method-name]([method-signature])": {
-          "[parameter-name]": {
-            "[attribute-name]": "[attribute-value]"
-          }
-        }
-      }
-    }
-  }
-}
+### Custom attribute on type field
+
+```xml
+<linker>
+  <assembly fullname="Assembly">
+    <type fullname="Assembly.A">
+      <field name="MyNumericField">
+        <attribute fullname="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembers">
+          <argument>DefaultConstructor</argument>
+        </attribute>
+      </field>
+    </type>
+  </assembly>
+</linker>
+```
+
+### Custom attribute on property field
+
+```xml
+<linker>
+  <assembly fullname="Assembly">
+    <type fullname="Assembly.A">
+      <property name="MyNumericField">
+        <attribute fullname="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembers">
+          <argument>DefaultConstructor</argument>
+        </attribute>
+      </property>
+    </type>
+  </assembly>
+</linker>
+```
+
+### Custom attribute on method
+
+```xml
+<linker>
+  <assembly fullname="Assembly">
+    <type fullname="Assembly.A">
+      <method signature="System.Void Method1(System.Boolean)">
+        <parameter name="boolParameter">
+          <attribute fullname="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembers">
```

Comment:
```suggestion
      <method signature="System.Void Method1(System.Type)">
        <parameter name="typeParameter">
          <attribute fullname="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembers">
```

Reply:
The sample should be realistic - we don't allow (rather ignore) these attributes on non-type members.